src/workspaces/cookiefactoryv2/cdk/lib/cookiefactory_v2_stack.ts

```diff
@@ -374,6 +377,120 @@ export class CookieFactoryV2Stack extends cdk.Stack {
         if (`${this.stackName}`.length > 32) {
             throw Error('stackName too long: stackName is used in some generated resource names with length restrictions, please use a stackName no longer than 32 characters')
         }
+        // Create Cognito Resources: User Pool, User Client, User, Identity Pool 
+
+        // todo: figure out why everything except UserPool gets deleted during cdk destroy
+        const userPool = new cognito.UserPool(this, "CookiefactoryUserPool", {
+            userPoolName: "CookiefactoryUserPool",
+            selfSignUpEnabled: false, 
+            signInAliases: {
+                email: true,
+            },
+            autoVerify: {
+                email: false,
+            },
+            accountRecovery: cognito.AccountRecovery.EMAIL_ONLY,
+            email: cognito.UserPoolEmail.withCognito(),
+            mfa: cognito.Mfa.OFF,
+            passwordPolicy: {
+                minLength: 8,
+                requireLowercase: true,
+                requireUppercase: true,
+                requireDigits: true,
+                requireSymbols: true,
+                tempPasswordValidity: cdk.Duration.days(7),
+              },
+            deletionProtection: false,
+            removalPolicy: cdk.RemovalPolicy.DESTROY,
+        });
+
+        // todo: turn off all the Hosted UI configurations; note status is unavailable so this is low priority
```

Comment:
just curious on details of this TODO, what's the impact of the Hosted UI config + what's necessary to fix it?

Reply:
There is no impact actually, was just trying to align exactly with the console steps. Given that I could not find an explicit parameter for the config in the CDK docs, to "fix" it would be to create a custom aws resource (similar to the set-admin-pass we talked about). I think I'll remove TODO as it has no impact on current experience